scripts/stop_local.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
```

Comment:
This seems to require an argument, but the usage text indicates the argument is optional